gridstatus/ercot.py

```diff
@@ -3807,14 +3807,46 @@ def _process_cop_adjustment_period_snapshot_60_day_data(
             .reset_index(drop=True)
         )
 
-        # Columns not in older data files. The columns first have data on 2024-06-28
-        for col in ["Minimum SOC", "Maximum SOC", "Hour Beginning Planned SOC"]:
+        # Columns not in older data files or not in newer data files.
```

Comment:
I didn't check far enough back on the initial implementation. The updates will work back to the beginning of the dataset.